lib/public/views/Logs/Logs.js

```diff
@@ -50,6 +53,70 @@ export default class Overview extends Observable {
         this.notify();
     }
 
+    /**
+     * Create the log with the variables set in the model, handle errors appriopately
+     * @returns {undefined}
+     */
+    async createLog() {
+        const response = await fetchClient('api/logs', {
+            method: 'POST',
+            headers: {
+                Accept: 'application/json',
+                'Content-Type': 'application/json',
+            },
+            body: JSON.stringify({
+                title: this.getTitle(),
+                text: this.getText(),
+            }),
+        });
+
+        const result = await response.json();
+
+        if (!(result === undefined)){
+            this.error = false;
+            this.model.router.go('/');
```

Comment:
Is the expected behaviour to go to the main page or to the detail view (where an user can add tags / attachemtens / ... )?

Reply:
This feature has been added